src/frontends/sql/testing/operation_view_utils.h

```diff
@@ -25,7 +25,15 @@ std::optional<uint64_t> ExtractIdxAfterPrefix(
 // trailing numeric suffixes being translated into positions in the resulting
 // vector. If the density assumption is not met, this method asserts.
 std::vector<ir::Value> GetVecWithPrefix(const ir::NamedValueMap &map,
-                                               absl::string_view prefix);
+                                        absl::string_view prefix);
+
+// This method assumes that, within the given `NamedValueMap`, there are a
+// subset of elements that start with some prefix `prefix` and end with a key
+// that should be associated with that value in the resulting map. It
+// composes that map and returns it.
+absl::flat_hash_map<std::string, ir::Value> GetMapWithPrefix(
```

Comment:
`GetElementsWithPrefix` or `GetMapEntriesWithPrefix` might be a better name?

Reply:
Went with `GetMapEntriesWithPrefix`.